src/Kernel/Magic/TraceMagic.cs

```diff
@@ -15,19 +15,20 @@ namespace Microsoft.Quantum.IQSharp.Kernel
 {
     /// <summary>
     ///      Contains the JSON representation of the <see cref="ExecutionPath"/>
-    ///      and the ID of the HTML div that will contain the visualization of the
-    ///      execution path.
+    ///      and metadata used in the visualization of the execution path.
     /// </summary>
     public class ExecutionPathVisualizerContent : MessageContent
     {
         /// <summary>
         ///     Initializes <see cref="ExecutionPathVisualizerContent"/> with the
-        ///     given <see cref="ExecutionPath"/> (as a <see cref="JToken"/>) and HTML div ID.
+        ///     given <see cref="ExecutionPath"/> (as a <see cref="JToken"/>), HTML div ID,
+        ///     and the visualization style of the output circuit.
         /// </summary>
-        public ExecutionPathVisualizerContent(JToken executionPath, string id)
+        public ExecutionPathVisualizerContent(JToken executionPath, string id, string style)
         {
             this.ExecutionPath = executionPath;
             this.Id = id;
+            this.Style = style;
```

Comment:
If you make `style` an enum type, it'd be reasonable to just `.ToString()` it here and keep the message content property `Style` as a string.